modules/bloop-rifle/src/main/scala/scala/build/blooprifle/internal/Operations.scala

```diff
@@ -349,6 +349,12 @@ object Operations {
           stop0,
           interactiveSession = false
         )
+        if (retCode != 0)
+          logger.error(
+            s"""Something is wrong with the current configuration. Nailgun server failed with exit code $retCode.
+               |To resolve the problem, please run the ${Console.BOLD}clean${Console.RESET} sub-command to clear the working directory and remove caches.
+               |If the error persists, please report the issue as a bug and attach a log with increased verbosity by passing ${Console.BOLD}-v -v -v${Console.RESET}.""".stripMargin)
```

Comment:
I'd change the phrasing a bit here
```suggestion
            s"""Bloop bsp command exited with code $retCode. Something may be wrong with the current configuration.
               |Running the ${Console.BOLD}clean${Console.RESET} sub-command to clear the working directory and remove caches might help.
               |If the error persists, please report the issue as a bug and attach a log with increased verbosity by passing ${Console.BOLD}-v -v -v${Console.RESET}.""".stripMargin)
```